seqr/views/apis/data_manager_api.py

```diff
@@ -395,6 +396,69 @@ def load_rna_seq_sample_data(request, sample_guid):
     return create_json_response({'success': True})
 
 
+def _load_phenotype_prioritization(file_path, user):
+    data_by_id = load_phenotype_prioritization_data_file(file_path)
```

Comment:
it would be better to use a more descriptive name, maybe `data_by_project_sample_id`